### PR TITLE
feat: first version of the app

### PR DESCRIPTION
**Pull request by Guille-lg**
# Summary

This pull request contains the first version of the app, with basic functionality to label different sdg paragraphs.

### CODE REVIEW

File: assets/UNDP_logo.svg.png
Comment (mykolaskrynnyk) [inline assets/UNDP_logo.svg.png:1]: @Guille-lg , the official emblem should not be present either in the repository or in the application.

File: scripts/components.py [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+        className='app-header',
+        height=120,
+        withBorder=True,
+        children=[
+            dmc.Image(
+                src='/assets/UNDP_logo.svg.png',
+                className='app-header-logo',
+                height=100,
+                width=50
+            ),
```
Comment (mykolaskrynnyk) [inline scripts/components.py]: Do not use the official emblem in the app.

File: scripts/database.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+            doc['_id'] = str(doc['_id'])
+            if doc['_id'] not in doc_ids and check_queue(doc['_id']):
+                update_queue(doc['_id'])
+                doc_ids.append(doc['_id'])
+                return doc, doc_ids
+    
+    raise Exception('No documents found')
+    
+def update_queue(_id):
+    load_dotenv()
```
Comment (mykolaskrynnyk) [inline scripts/database.py]: No need to use `load_dotenv` inside functions. Just include this call in the `app.py`. The env variables will then be available throughout the application anyway.

File: scripts/database.py [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    collection = db['paragraph_queue']
+
+    doc = collection.find_one({'_id': _id})
+    if doc:
+        collection.replace_one({'_id': _id}, {'_id' : _id, 'date' : datetime.now()})
+    else:
+        collection.insert_one({'_id' : _id, 'date' : datetime.now()})
+
+def check_queue(_id):
+    load_dotenv()
```
Comment (mykolaskrynnyk) [inline scripts/database.py]:
```suggestion
```

**Comment by Guille-lg** [comment]
@mykolaskrynnyk fixed the issues with the logo and requirements file, ready for review

**Comment by Guille-lg** [comment]
@mykolaskrynnyk fixed the issues with the logo and requirements file. For some reason, the requirements file commit still shows the file as binary, but when checking out the branch the file is fine. May need additional review.

File: assets/UNDP_logo.svg.png
Comment (mykolaskrynnyk) [inline assets/UNDP_logo.svg.png:1]: @Guille-lg , this can be removed now too.

**Review by mykolaskrynnyk — CHANGES_REQUESTED** [review]
@Guille-lg , once the comments are resolved, the request will be merged.

**Comment by Guille-lg** [comment]
Fixed the load_dotenv and UNDP logo issues

**Pull request merged**